about.html

```diff
@@ -7,67 +7,89 @@
+        
+        <h2>Strategy</h2>
+        
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
+        <button>Learn More</button>
+      </section>
 
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+      <section class="cta-content">
+        <img src="img/about-working.png" alt="strategy">
+        
+        <h2>How We Work</h2>
+        
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    Learn More
+        <button>Learn More</button>
+      </section>
 
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+      <section class="cta-content">
+        <img src="img/about-office.png" alt="strategy">
+        
+        <h2>Places We Work</h2>
 
-    Learn More
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    Let's Work Together
+        <button>Learn More</button> 
+      </section>
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+      <section class="cta-content">
+        <img src="img/about-meeting.png" alt="strategy">
         
-      <footer>
-        <nav>
-          <a href="index.html">Home</a>
-          <a href="#">About</a>
-          <a href="#">Products</a>
-          <a href="#">Blog</a>
-          <a href="#">Contact</a>
-        </nav>
-      </footer>
-    </div><!-- container -->        
+        <h2>Collaboration</h2> 
+        
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+
+        <button>Learn More</button>
+      </section>
+    </section>
+
+    <section class="bottom-content">
+      <h2>Let's Work Together</h2>
+
+      <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+    </section>
+
+    <footer>
+      <nav>
+        <a href="index.html">Home</a>
+        <a href="#">About</a>
+        <a href="#">Products</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+      </nav>
+    </footer>
+  </div><!-- container -->        
 </body>
 </html>
```

Comment:
remember to link up all the navigation a tags in both headers and footers to make navigating between the pages easy